docs/introduction.md

```diff
@@ -85,3 +85,72 @@ Available download clients and actions
 The development of autobrr started in Early 2020, entering rapid development in Summer 2021 due to dissatisfaction with needing 3+ tools to do one job. Autobrr has since gained quite a bit of traction and has a growing community supporting the project.
 
 Autobrr was developed with resource consumption in mind. The software uses API calls to reduce unnecessary downloads of .torrent files from sites like BTN, RED, PTP, and GGn. On other sites, it will download the .torrent only if the information is not present in the announce message.
+
+## Quick Start
+
+The following considers a common use case, feeding IRC announcements for a private
+tracker to a Servarr instance, as a way to illustrate how the components of autobrr fit
+together and what is required to get up and running.  It's not the only use case and
+other use cases may require more RTFM'ing in [the configuration docs](./configuration).
+
+1. [Install autobrr](./installation/linux).
+
+	Proceed once the web UI is accessible.
+
+2. Create a user for yourself.
+
+	Most easily done through the initial "wizzard" in the web UI when autobrr is first
+	started.
+
+3. [Register a nick on your indexer's IRC
+   network](https://autobrr.com/configuration/irc#registering-with-nickserv).
+
+4. [Group a "bot" nick with your real nick](./configuration/irc#grouping-nicks)
+
+5. Add an [indexer](./configuration/indexers).
+
+6. Add a [Servarr instance](./configuration/download-clients#sonarr)
```

Comment:
This should probably just be 

6. Add a [Download client](./configuration/download-clients)

Reply:
That change might better fit the terminology of autobrr, but it would be less clear for someone coming in for this use case.  IOW, if I neutered this of that specific use case it would become much longer and I wouldn't be able to write it.  But if someone else agrees with you, I'll make the change.

Reply:
Im with gl0ryus, as I don't think a quick start guide should be focused specifically towards the arrs. And the use of "Servarr instance" would create more confusion for everybody else I think.

I get your point though!

Reply:
Ok, I'll make the change to these specific parts of the language.  But note that much else of this Quick Start is specific to this Servarr use case.  And I'm not qualified to write any other version of a quick start.

Reply:
> 6\. (qBittorrent, Sonarr, Radarr etc)

I think the note following covers that, no?